lib/macho/load_commands.rb

```diff
@@ -148,7 +148,11 @@ module MachO
   # represented, and no actual data. Used when a more specific class
   # isn't available/implemented.
   class LoadCommand < MachOStructure
-    # @return [Fixnum] the offset in the file the command was created from
+    # @return [MachO::MachOView] the raw view associated with the load command
+    attr_reader :view
+
+    # @return [Fixnum] the load command's offset in the source file
+    # @deprecated use {#view} instead
     attr_reader :offset
```

Comment:
I have somehow managed to never use `Forwardable`, but it looks like it could make a lot of `ruby-macho`'s delegation a lot simpler. I'm going to make a note of it, but go with the `def` until I know it well enough to roll it out everywhere :smile:.